python/gherkin3/token_matcher.py

```diff
@@ -50,13 +91,11 @@ def match_StepLine(self, token):
             title = token.line.get_rest_trimmed(len(keyword))
             self._set_token_matched(token, 'StepLine', title, keyword)
             return True
-
         return False
 
     def match_Comment(self, token):
         if not token.line.startswith('#'):
             return False
-
```

Comment:
This whole PR seems to be change for the sake of change. Is there anything substantial I'm missing? Like a bugfix?